model_analyzer/analyzer.py

```diff
@@ -109,14 +109,22 @@ def profile(self, client, gpus):
             self._metrics_manager.profile_server()
             self._server.stop()
 
-        # Profile each model, save state after each
-        for model in self._config.profile_models:
-            if self._state_manager.exiting():
-                break
+        # Profile all models concurrently
+        if self._config.run_config_profile_models_concurrently_enable:
             try:
-                self._model_manager.run_model(model=model)
+                self._model_manager.run_models(
```

Comment:
why is this in a try block?

Reply:
I don't know? I just copied what already existed